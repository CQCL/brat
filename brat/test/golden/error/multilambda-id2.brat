g(Nat) -> { Qubit -o Qubit }
g = { 0 => ||succ(n) => | }
