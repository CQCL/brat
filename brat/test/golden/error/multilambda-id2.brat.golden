Error in test/golden/error/multilambda-id2.brat@2:12-13
g = { 0 => ||succ(n) => | }
           ^

  Elab error "Noun required at this position"

